.github/workflows/ci.yml

```diff
@@ -10,6 +10,7 @@ env:
   BAAS_API_KEY: ${{ secrets.ATLAS_QA_PUBLIC_API_KEY }}
   BAAS_PRIVATE_API_KEY: ${{ secrets.ATLAS_QA_PRIVATE_API_KEY }}
   BAAS_PROJECT_ID: ${{ secrets.ATLAS_QA_PROJECT_ID}}
+  BAAS_JWKS_URL: https://raw.githubusercontent.com/realm/realm-dart/${{ github.event_name == 'pull_request' && github.head_ref || github.ref_name }}/test/data/jwt_keys/jwks.json
```

Comment:
I feel this is a bit of an overkill. We can test JWT auth with the public key only.

Reply:
Why not to have it?

Reply:
OK, removed.